### ISSUE TITLE
Error encountered using custom JSON file on AGE System

### ISSUE THREAD

**Issue by lupestro** [League-of-Foundry-Developers/torch#26]
Hey @lupestro I was using a custom JSON file before the last update and it was working fine, but now I keep having errors on `library.js` in the exactly the same portion of code that was changed. My errors are normally between lines 25 and 34. Here is the json code I was using.

*Note:* I am using module on [AGE System](https://foundryvtt.com/packages/age-system) and FVTT 10.286.

```
{
    "age-system": {
      "system": "age-system",
      "topology": "standard",
      "quantity" : "quantity",
      "sources": {
        "Lanterna": {
          "name": "Lanterna",
          "type": "equipment",
          "consumable": false,
          "states": 2,
          "light": [
            { 
              "bright": 10, "dim": 15, "angle": 30, "color": "#ff9329", "alpha": 0.5,
              "animation": { "type": "torch", "speed": 5, "intensity": 5, "reverse": false } 
            }
          ]
        },
        "Visão Noturna": {
            "name": "Visão Noturna",
            "type": "equipment",
            "consumable": false,
            "states": 2,
            "light": [{
                "bright": 1000, "dim": 1000, "angle": 360, "color": "#184601", "alpha": 0.4,
                "animation": { "type": "none", "speed": 5, "intensity": 5, "reverse": false },
                "light": {"luminosity": 0.3}
            }]
        }
      }
    }
}
```
Attached is the error I get when rendering token hud:
![image](https://user-images.githubusercontent.com/66521243/193369785-b1dcf699-23b3-4201-906f-b628820cf2ce.png)

I am logging this here as I believe the error I am facing is closely related to the ones posted in this thread.

Cheers!

_Originally posted by @vkdolea in https://github.com/League-of-Foundry-Developers/torch/issues/23#issuecomment-1264132855_

**Comment by lupestro** [comment]
@vkdolea Good catch. I had missed an important edge case. I have a one-line fix but can you send me a character to use with this so I can watch the light going on and off?

**Comment by lupestro** [comment]
This should be fixed in Torch v2.1.3.
